packages/mds-db/schema.ts

```diff
@@ -222,15 +304,18 @@ const TABLE_COLUMNS: { [T in TABLE_NAME]: Readonly<COLUMN_NAME[]> } = {
 const TABLE_KEY: { [T in TABLE_NAME]: COLUMN_NAME[] } = {
   [TABLE.attachments]: [COLUMN.attachment_id],
   [TABLE.audit_attachments]: [COLUMN.attachment_id, COLUMN.audit_trip_id],
-  [TABLE.audits]: [COLUMN.audit_trip_id],
   [TABLE.audit_events]: [COLUMN.audit_trip_id, COLUMN.timestamp],
+  [TABLE.audits]: [COLUMN.audit_trip_id],
   [TABLE.devices]: [COLUMN.device_id],
   [TABLE.events]: [COLUMN.device_id, COLUMN.timestamp],
   [TABLE.geographies]: [COLUMN.geography_id],
   [TABLE.geography_metadata]: [COLUMN.geography_id],
   [TABLE.migrations]: [COLUMN.migration],
   [TABLE.policies]: [COLUMN.policy_id],
   [TABLE.policy_metadata]: [COLUMN.policy_id],
+  [TABLE.reports_device_states]: [COLUMN.timestamp, COLUMN.device_id, COLUMN.provider_id, COLUMN.type],
+  [TABLE.reports_providers]: [COLUMN.provider_id, COLUMN.vehicle_type, COLUMN.start_time],
+  [TABLE.reports_trips]: [COLUMN.provider_id, COLUMN.device_id, COLUMN.trip_id],
```

Comment:
Seems these PKs may have more columns that absolutely necessary for uniqueness, but perhaps there are some query benefits to including the extra columns?

Reply:
I tried to think how we would access the data most often with what fields. Forcing it's uniqueness past what's necessary will give us better read performance.